R/geo.R

```diff
@@ -115,6 +143,32 @@ addGeoMetadata <- function (variable) {
     return(match_geo)
 }
 
+# format the error message when there are multiple matches
+multiMatchHelper <- function (match_scores, match_field, quoted_var) {
+    msg <- c("There is more than one possible match. You will need to specify ", 
+             "the geography manually using one of the following ",
+             "CrunchGeographies:\n")
+    
+    calls <- apply(match_scores, 1, function(match_row) {
+        # TODO: add coloring to this message when crayon is available
```

Comment:
Yes, I was just looking at bringing in crayon for the variable folder stuff I've started. FWIW it's a simple addition, only depends on base packages and doesn't require compilation: https://github.com/r-lib/crayon/blob/master/DESCRIPTION#L36